utils/detect-standard.ts

```diff
@@ -0,0 +1,34 @@
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts'
+
+import { InterfaceId } from '@/types/assets'
+
+export const detectStandard = async (
+  contractAddress: Address
+): Promise<InterfaceId | undefined> => {
```

Comment:
Can this be of any help?

https://docs.lukso.tech/tools/erc725js/classes/ERC725#supportsinterface

Reply:
Sure, I didn't know about this function!